src/bp/core/server.ts

```diff
@@ -220,6 +220,9 @@ export default class HTTPServer {
      */
     this.app.use((req, res, next) => {
       res.header('X-Powered-By', 'Botpress')
+      Object.entries(config.headers).map(header => {
+        res.setHeader(header[0], header[1])
+      })
```

Comment:
This will fail if headers are undefined in the config
Also, shouldn't they be set as default headers instead, so we avoid looping through them for every incoming request ?